logging/dedicated-efk-logging.adoc

```diff
@@ -0,0 +1,65 @@
+:context: dedicated-efk-logging
+[id="dedicated-efk-logging"]
+= About cluster logging and {product-title}
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+As an {product-title} cluster administrator, you can deploy cluster logging
+to aggregate logs for a range of services.
+
+{product-title} clusters come configured with logging via `ElasticSearch`
+operators installed and `Curator` pre-installed to have a two-day retention of
+logs. To utilize these features, create a `ClusterLogging` Custom Resource in
+the `openshift-logging` namespace with the following `.yaml` information:
```

Comment:
s/with/that contains

In this context it's YAML, not .yaml

Reply:
Fixed!